uw-frame-static/index.html

```diff
@@ -15,11 +15,20 @@
   <link ng-href="css/themes/uw-{{portal.theme.name}}.css" rel="stylesheet" type="text/css"/>
   <link href="my-app/my-app.css" rel="stylesheet" type="text/css"/>
   <link rel="shortcut icon" href="bower_components/uw-ui-toolkit/dist/img/favicon.ico" type="image/x-icon"/>
+  <link rel="import" href="static.html" />
```

Comment:
We could add in https://github.com/webcomponents/webcomponentsjs if we wanted to support all browsers, but by that time the loading splash will be worthless. I guess we could just copy/paste in the `static.html` into this index.html page. Was just trying to avoid dup code.